packages/animation-curve-block/src/components/CardText.tsx

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { RichTextEditor, convertToRteValue, joinClassNames } from '@frontify/guideline-blocks-shared';
+import { TextStyles } from '@frontify/fondue';
+
+import { CardTextProps } from '../types';
+import { getDescriptionPlugin, getTitlePlugin } from '../helpers';
+
+export const CardText = ({
+    appBridge,
+    title,
+    description,
+    hasBorder,
+    designTokens,
+    updateValueOnChange,
+    isEditing,
+    setTitle,
+    setDescription,
+}: CardTextProps) => {
+    return (
+        <div className={joinClassNames([hasBorder && 'tw-px-4', 'tw-pt-4 tw-flex-1'])}>
+            <RichTextEditor
```

Comment:
As its within the [Figma](https://www.figma.com/file/sCEfTYxZNNKlhrpW7cCoct/New-Content-Block---Animation-curve?node-id=2133-433127&t=v8WJ8blA56sERIOK-0) i assume it should be a RTE.

And it also doesn't break the layout as its just displayed below (no alignments and so on, not like in the do & dont), i think we should discuss that with a designer...

Reply:
If the h3 design token has margin-top I think it would add the extra space to top. Behaviours like this were considered bugs for the Do & Don't.

e.g.
https://frontify.slack.com/archives/CCXT7AFD0/p1681807431078459

Reply:
Yeah but that was a bug because it wasn't aligned with the icon anymore, in this case its always below and i wouldn't consider it as bug as it just add more spacing (like defined)